requirements.txt

```diff
@@ -23,4 +23,4 @@ fastly==0.0.2
 pandas==0.17.1
 BeautifulSoup==3.2.1
 boto3==1.2.5
-syncano==4.2.0
+git+https://github.com/Syncano/syncano-python.git@develop
```

Comment:
We should push 5.0 asap to PyPI and get rid of it. Or at very least make a stable/5.0.x branch out of it.
@opalczynski poke! ;)